### ISSUE TITLE
Confusing with gitea version/naming

### ISSUE THREAD

**Issue by sentabi** [go-gitea/gitea#2275]
I'm confusing with gitea version, for example : 

Gitea 1 
```
gitea.personal.com
Gitea Version: 27798c3
```
Gitea 2
```
gitea.company.com
Gitea Version: 2b05b10
```
try.gitea.io 
```
Gitea Version: 27798c3
```
which version is `27798c3` or `2b05b10`? 

I know `2b05b10` is from https://dl.gitea.io/gitea/master/ because I installed that yesterday.

Another one : If I use master version I've menu to add GPG key, but not in stable version ex 1.1.3
I use gitea from 1.0, if that matter with this behaviour. 

If I using stable version, the menu is horizotal, 
![gitea version 1 1 3](https://user-images.githubusercontent.com/203196/29055922-af00c006-7c29-11e7-94b4-9611201bb3a0.png)
master version : menu vertical
![gitea version 27798c3](https://user-images.githubusercontent.com/203196/29055939-cb13e58e-7c29-11e7-8d12-ea79776fa9eb.png)

**Comment by lunny** [comment]
please see https://github.com/go-gitea/gitea/blob/master/CONTRIBUTING.md#versions

**Comment by bkcsoft** [comment]
`master` is the absolute latest and does not have a version number. If you need the same version you should use fixed versions (v1.1.3 being the latest).

**Comment by tboerger** [comment]
But @sentabi is right, the version just shows the Git commit and not the correct version.
